addcomplete.php

```diff
@@ -0,0 +1,40 @@
+<?php
+require_once 'head.html';
+require_once 'Tree.php';
+require_once 'TreeDBConnect.php';
+
+$tree = new TreeDBConnect();
+
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+    $name = $_POST['name'];
+    $scientific_name = $_POST['scientific_name'];
+    $leaf_type = $_POST['leaf_type'];
+    $leaf_colour = $_POST['leaf_colour'];
+    $flower_colour = $_POST['flower_colour'];
+    $image = $_POST['image'];
+}
+
+if ($_POST['name'] === '' || $_POST['scientific_name']
+    === '' || $_POST['leaf_type']
+    === '' || $_POST['leaf_colour']
+    === '' || $_POST['flower_colour']
+    === '' || $_POST['image'] === '') {
```

Comment:
Since you make the variables containing the post data above, would make sense to use those variables here rather than going back into $_POST

Reply:
I've replaced with the variables
Makes sense when you say it out loud :)